R/csv-utils.R

```diff
@@ -1,19 +1,42 @@
-#' @keywords internal
-mrgda_read_csv <- function(.file) {
+#' Read csv file
+#'
+#' @description
+#' Reads csv files and formats the data to replace any "." with
+#' NA values.
+#'
+#' @param .file file path to csv file
+#'
+#' @export
+read_nm_csv <- function(.file) {
 
-  data.table::fread(
-    file = .file,
-    sep = ",",
-    quote = FALSE,
-    na.strings = "."
-  ) %>%
+  df <-
+    data.table::fread(
+      file = .file,
+      sep = ",",
+      quote = FALSE,
+      na.strings = "."
+    ) %>%
     as.data.frame() %>%
     suppressMessages()
 
+  cli::cli_alert_success(paste0("Successfully read: '", .file, "'"))
+  cli::cli_alert_info('"." replaced with NA')
+
+  df
 }
 
-#' @keywords internal
-mrgda_write_csv <- function(.data, .file) {
+
+#' Write csv file
+#'
+#' @description
+#' Writes csv files and formats the data to replace any NA with
+#' "." values.
+#'
+#' @param .data data.frame
+#' @param .file file path to write out csv file
+#'
+#' @noRd
+write_nm_csv <- function(.data, .file) {
```

Comment:
Again, I'd let users pass anything through, adding sensible defaults to `write_nm_csv()` signature. 

For example, pass `quote` through, with default value `FALSE`.